src/explore-education-statistics-common/src/modules/find-statistics/components/charts/__tests__/__snapshots__/MapBlock.test.tsx.snap

```diff
@@ -0,0 +1,285 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MapBlock MapBlock renders 1`] = `
+<div>
+  <div
+    class="govuk-grid-row"
+  >
+    <div
+      aria-live="assertive"
+      class="govuk-grid-column-one-third"
+    >
+      <form>
+        <div
+          class="govuk-form-group govuk-!-margin-bottom-6"
+        >
+          <label
+            class="govuk-label"
+            for="selectedIndicator"
+          >
+            Select data to view
+          </label>
+          <select
+            class="govuk-select select"
+            id="selectedIndicator"
+            name="selectedIndicator"
+          >
+            <option
+              value="28"
+            >
+              Authorised absence rate
+            </option>
+            <option
+              value="26"
+            >
+              Overall absence rate
+            </option>
+            <option
+              value="23"
+            >
+              Unauthorised absence rate
+            </option>
+          </select>
+        </div>
+        <div
+          class="govuk-form-group govuk-!-margin-bottom-6"
+        >
+          <label
+            class="govuk-label"
+            for="selectedLocation"
+          >
+            Select a location
+          </label>
+          <select
+            class="govuk-select select"
+            id="selectedLocation"
+            name="selectedLocation"
+          >
+            <option
+              value="E92000001"
+            >
+              England
+            </option>
+            <option
+              value=""
+            >
+              select...
+            </option>
+          </select>
+        </div>
+      </form>
+      
+      <div>
+        <h3
+          class="govuk-heading-s"
+        >
+          Key to 
+          Authorised absence rate
+        </h3>
+        <dl
+          class="govuk-list"
+        >
+          <dd
+            class="legend"
+          >
+            <span
+              class="rate0"
+            >
+               
+            </span>
+             
+            5.0
```

Comment:
Similar to the `DataBlock renders map` test case, are we supposed to have multiple rates that are all the same?

Reply:
Added a better test to reflect what the API should be returning.